.travis.yml

```diff
@@ -23,60 +25,6 @@ stages:
 matrix:
   fast_finish: true
   include:
```

Comment:
Hey

As it is now, acceptance tests will not be executed on travis. To include that, we need to update .sync file and re-run pdk update

Cheers!